spec/v1.0/envelope.md

```diff
@@ -1,38 +1,12 @@
 # Envelope layer specification
 
-Version: v1.0
+Version: [DSSE v1.0]
 
 The Envelope is the outermost layer of the attestation, handling
-authentication and serialization. The format and protocol are defined in
-[DSSE] and adopted by in-toto in [ITE-5].
+authentication and serialization.
 
-## Schema
+The format and protocol are defined per [DSSE v1.0]. In the in-toto
+Attestation Framework, the DSSE payload is a base64-encoded JSON [Statement].
 
-```jsonc
-{
-  "payloadType": "application/vnd.in-toto+json",
```

Comment:
I think it's fine to refer people to the DSSE spec for the format, but what this change misses is that we no longer specify what the payloadType for an in-toto DSSE should be.  I.e. we probably do want to say

```
> Identifier for the encoding of the payload. Always
> `application/vnd.in-toto+json`, which indicates that it is a JSON object
> with a `_type` field indicating its schema.
```